<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvestigatorDialog</class>
 <widget class="QDialog" name="InvestigatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Investigator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="comboBoxHorizontalLayout">
     <item>
      <widget class="QLabel" name="investigatorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Investigator: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="investigatorComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="usernameHorizontalLayout">
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Username: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="firstNameHorizontalLayout">
     <item>
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lastNameHorizontalLayout">
     <item>
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Last Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="institutionHorizontalLayout">
     <item>
      <widget class="QLabel" name="institutionLabel">
       <property name="text">
        <string>Institution: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="institutionLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="eMailHorizontalLayout">
     <item>
      <widget class="QLabel" name="eMailLabel">
       <property name="text">
        <string>eMail: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eMailLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InvestigatorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InvestigatorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>InvestigatorDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
